code/python/example.py

```diff
@@ -1,22 +1,45 @@
-# pip install neo4j-driver
-# python example.py
+# pip install neo4j
 
-from neo4j import GraphDatabase, basic_auth
+from neo4j import (
+    GraphDatabase,
+    basic_auth,
+)
 
-driver = GraphDatabase.driver(
-    "bolt://<HOST>:<BOLTPORT>", 
-    auth=basic_auth("<USERNAME>", "<PASSWORD>"))
-session = driver.session()
+scheme = "neo4j"
+port = 7687
+host_name = "example.com"
+user_name = "example_user"
+user_password = "example_password"
 
-cypher_query = '''
-MATCH (movie:Movie)<-[:ACTED_IN]-(actor)-[:ACTED_IN]->(rec:Movie)
-WHERE movie.title IN $favorites
-RETURN rec.title as title, count(*) as freq 
-ORDER BY freq DESC LIMIT 5
-'''
+uri = "{scheme}://{host_name}:{port}".format(scheme=scheme, host_name=host_name, port=port)
 
-results = session.run(cypher_query,
-  parameters={"favorites":["The Matrix"]})
+driver = GraphDatabase.driver(uri, auth=basic_auth(user_name, user_password))
 
-for record in results:
-  print(record['title'])
+def get_recomended_movies(transaction, favorites):
```

Comment:
By convention, we use `tx ` instead of `transaction`. This is also better for restricted screen space for example code!